tools/src/test/java/org/apache/kafka/tools/FeatureCommandTest.java

```diff
@@ -47,22 +47,16 @@
 @ClusterTestDefaults(clusterType = Type.KRAFT)
```

Comment:
It seems we don't need `ClusterTestDefaults` for this test class, since all test cases have defined that.

Reply:
oh you are right, I didn't notice that part. Just remove it!